docs/raspberry-pi5-ssd-install-instructions.md

```diff
@@ -0,0 +1,47 @@
+# How to install on a Raspberry Pi 5 with PCI-e SSD
+
+## Appropriate Hardware
+
+The following guide has been tested using a Pi5/8GB and a GeeekPi P33 PoE+PCI-e HAT. (See: https://pipci.jeffgeerling.com/hats/geeekpi-p33-m2-nvme-poe-hat.html )
+
+Other HAT's should work just fine as long as they are supported by the Pi.
+
+We used a 256GB M.2 NVMe SSD as it came in the 2242 form factor, and was small and cheap (at time of writing, these were approx £15+vat from our supplier).
+
+You may also need a SD Card initially to configure your Pi, once the first steps are done you won't require this.
+
+## Install procedure
+
+Using the Raspberry Pi Imager and appropriate USB adapters, write the 64-bit version of Raspberry Pi OS to both a SD card and the SSD.
+
+There are other ways (Such as a network boot if enabled on the Pi, Booting from SD and using the SD Card copier utility to copy the OS to the SSD, or Booting from SD and using the RPI Imager to write a fresh copy to the SSD, or using rpiboot mode to present the SSD as mass storage to a PC.
+
+Do whatever is easiest for you!
+
+### Boot from SD
+
+Depending on the bootloader version of your Pi, you may need to boot from SD first and set the bootloader to boot from PCIe.
+
+- Once booted, run the RPI EEPROM configurator: `sudo rpi-eeprom-config -edit`
```

Comment:
```suggestion
- Once booted, run the RPI EEPROM configurator: `sudo rpi-eeprom-config -edit`. This will open up the [Nano](https://www.nano-editor.org/) text editor.
```